modules/reference/pages/feature/sessionDatabase-1.0/examples.adoc

```diff
@@ -0,0 +1,22 @@
+== Examples
+
+=== Minimum configuration to persist session data to a database
+
+The following example shows the minimum configuration for database persistence:
+
+[source, java]
+----
+<fileset id="DerbyFiles" includes="*.jar" dir="server1/derby/client"/>
```

Comment:
here and line 14, need to remove "server1" per convo here: https://github.com/OpenLiberty/docs/issues/2867#issuecomment-720805100

suggest:
`dir="derby/client"/>`